contracts/modules/staking/StakingPool.sol

```diff
@@ -682,6 +703,7 @@ contract StakingPool is IStakingPool, Multicall {
       }
 
       uint expirationBucket = Math.divCeil(block.timestamp + request.period, BUCKET_DURATION);
+
       uint rewardStreamPeriod = expirationBucket * BUCKET_DURATION - block.timestamp;
```

Comment:
@shark0der do you see any concerns here with maintaining rewards streaming  the exact same, meaning, in the case of an edit the rewards are streamed based on the extra premium and just added to the rewards stream respecting the new end time of the edited segment as such:

https://github.com/NexusMutual/smart-contracts/blob/ad60a3a9815373b6b6c002ab7255ea7b2dd99751/contracts/modules/staking/StakingPool.sol#L705

```uint _rewardPerSecond = (premium * request.rewardRatio / REWARDS_DENOMINATOR) / rewardStreamPeriod;```

I heard there might be concerns regard APY in some way?